front/src/modules/timesStops/helpers/formatScheduleData.ts

```diff
@@ -0,0 +1,16 @@
+import { ISO8601Duration2sec, datetime2sec, secToHoursString } from 'utils/timeManipulation';
+
+import type { ScheduleEntry } from '../types';
+
+function formatScheduleData(schedule: ScheduleEntry, startTime: string) {
+  const arrival = schedule?.arrival ? ISO8601Duration2sec(schedule.arrival!) : null;
```

Comment:
Add an early return and remove the ? after schedule
```suggestion
    if (!schedule) return ''
    const arrival = schedule.arrival ? ISO8601Duration2sec(schedule.arrival!) : null;
```

Reply:
done